ETC.tsv

```diff
@@ -157,26 +157,26 @@ xETC_20150317_000156	Its slimy substance allows it to model itself into a protot
 ETC_20150317_000157	Rusrat
 xETC_20150317_000158	During the times of Zachariel the Great, these monsters were placed to stop intruders. Its name stems from grave robbers who encountered it.
 ETC_20150317_000159	Madakia
-xETC_20150317_000160	To fend off intruding grave robbers, court sorcerers created Madakia.
+xETC_20150317_000160	Court sorcerers created the Madakia to fend off intruding grave robbers.
 ETC_20150317_000161	Wheelen
 xETC_20150317_000162	In order to watch over sealed ruins that lacked an overseer, monsters were created with a diversity of magic. The wheelen is one of these monsters.
 ETC_20150317_000163	Wheelen 3
 ETC_20150317_000164	Mauros
-xETC_20150317_000165	The stones of the Royal Mausoleum form with magic into an ancient heavily armed knight.
+xETC_20150317_000165	The stones of the Royal Mausoleum magically assembled into an ancient, heavily armed knight.
 ETC_20150317_000166	Truffle
-xETC_20150317_000167	Though the reason is not known, this monster has an extreme hatred towards humans. It loves to attack them for food and survival.
+xETC_20150317_000167	This monster has an irrational and extreme hatred towards humans. It loves attacking them for food and survival.
 ETC_20150317_000168	Treepellia
 ETC_20150317_000169	Bramble
 ETC_20150317_000170	Bramble Ego
 ETC_20150317_000171	Willow
 ETC_20150317_000172	Willowna
 ETC_20150317_000173	Hanaming
-xETC_20150317_000174	The fallen hanaming leaves don't usually wither rapidly.
+xETC_20150317_000174	Fallen Hanaming leaves are long-lasting and don't wither rapidly.
```

Comment:
Avoid adding extra description to items especially when the context is unknown. The original translation only says they wither rapidly, not that they last long too. 
Change: Fallen Hanaming leaves are long-lasting and don't wither rapidly. -> Fallen Hanaming leaves usually don't wither rapidly.